helix-term/Cargo.toml

```diff
@@ -25,6 +25,11 @@ helix-core = { version = "0.3", path = "../helix-core" }
 helix-view = { version = "0.3", path = "../helix-view" }
 helix-lsp = { version = "0.3", path = "../helix-lsp" }
 
+tracing = "0.1"
+tracing-subscriber = "0.2"
+tracing-tracy = "0.6"
+tracing-flame = "0.1"
```

Comment:
https://docs.rs/tracing-tracy/0.6.1/tracing_tracy/#important-note

Both of these should be marked as optional and placed under separate features that aren't enabled by default.